kratos/python_scripts/gid_output_process.py

```diff
@@ -645,3 +653,48 @@ def __remove_list_files(self):
                         os.remove(f)
                     except WindowsError:
                         pass
+                    
+    #
+    def __remove_post_results_files(self, step_label):
```

Comment:
I agree with @josep-m-carbonell that we shouldn't stop this commit until we agree on a naming convention. I'm giving my OK and creating a new issue (#42) to continue the discussion.